### PR TITLE
Now summary and since not always present in COMMAND DOCS

### PR DESCRIPTION
**Pull request by enjoy-binbin**
see https://github.com/redis/redis/pull/10252
also added a example? the output may be a bit long though

### CODE REVIEW

File: commands/command-docs.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ You can use the optional _command-name_ argument to specify the names of one or
 The reply includes a map for each returned command.
 The following keys are always present in the reply:
 
-* **summary:** short command description (always present).
-* **since:** the Redis version that added the command (always present).
+* **summary:** short command description.
+* **since:** the Redis version that added the command.
 * **group:** the functional group to which the command belongs (always present).
```
Comment (oranagra) [inline commands/command-docs.md:11]: the fact it always exists, is an implementation detail, i suggest not to document that
```suggestion
* **group:** the functional group to which the command belongs.
```

**Review by oranagra — APPROVED** [review]

File: commands/command-docs.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ You can use the optional _command-name_ argument to specify the names of one or
 The reply includes a map for each returned command.
 The following keys are always present in the reply:
```
Comment (zuiderkwast) [inline commands/command-docs.md:7]: This heading is not correct anymore. I think we should remove it and merge the two lists of fields (always present and sometimes present).

Comment (oranagra) [reply commands/command-docs.md:7]: sorry, i was eager to merge it..
please just issue a new PR and merge it.

Comment (enjoy-binbin) [reply commands/command-docs.md:7]: ohh sorry. I did not go through the whole file and didn't see it. I will take care of it later

**Pull request merged**